### PR TITLE
docs(release): optimize the customize semver recipe

### PR DESCRIPTION
**Pull request by juristr**
found this one hard to parse/read so tried to give it more structure:
https://nx-dev-git-fork-juristr-docs-nx-release-customize-semver-nrwl.vercel.app/recipes/nx-release/customize-conventional-commit-types

### CODE REVIEW

**Review by isaacplmann — CHANGES_REQUESTED** [review]

File: docs/shared/recipes/nx-release/customize-conventional-commit-types.md [hunk 1]
```diff
@@ -1,112 +1,150 @@
 # Customize Conventional Commit Types
 
-Nx Release can defer to the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) standard to automatically determine the next version to release. To enable this behavior for versioning, see [Automatically Version with Conventional Commits](/recipes/nx-release/automatically-version-with-conventional-commits).
+[Nx release](/features/manage-releases) allows you to leverage the [conventional commits](/recipes/nx-release/automatically-version-with-conventional-commits) standard to automatically determine the next version increment.
 
-This recipe will cover how to customize the types of commits that trigger version bumps, how to customize the version bump for each type, and how to customize the changelog entry for each commit type.
+Nx follows the [conventional commit standard](https://www.conventionalcommits.org/en/v1.0.0/). By default, this results in:
```
[the remark this answers is not in the document]
Comment (juristr) [reply docs/shared/recipes/nx-release/customize-conventional-commit-types.md:5]: ah I should rephrase it.

File: docs/shared/recipes/nx-release/customize-conventional-commit-types.md [hunk 2]
```diff
@@ -1,112 +1,150 @@
...
 
-## Conventional Commits Usage within Nx Release
-
-The conventional commits configuration is used in two different places within Nx Release - once in the version step for determining the version bump, and once when generating changelogs.
-
-### Determine the Version Bump
```
[the remark this answers is not in the document]
Comment (juristr) [reply docs/shared/recipes/nx-release/customize-conventional-commit-types.md:11]: There's an exact identical one in the [recipe about configuring](https://nx.dev/recipes/nx-release/automatically-version-with-conventional-commits#determine-the-version-bump) semantic versioning with nx release. So I removed it here as it is more relevant on that one.

**Review by isaacplmann — APPROVED** [review]
